packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -164,29 +164,51 @@ const WorkflowStepStore = types
         // put is a MST method, not native to ES Map
         // the key is inferred from the identifier type of the target model
         const taskToStore = Object.assign({}, workflow.tasks[taskKey], { taskKey })
-        self.tasks.put(taskToStore)
+        try {
+          self.tasks.put(taskToStore)
+        }
+        catch (e) {
+          console.log(`${taskKey} ${taskToStore.type} is not a supported task type`)
```

Comment:
This would be better logged to sentry - packages/app-project/src/helpers/logger/logReactError.js

Reply:
Can that be used in the classifier?

Reply:
How does that error logger work without throwing an exception that causes the store to fail at the first unsupported task? The code here is supposed to store all supported tasks from the workflow, while skipping over unsupported tasks like the combo task (which is instead converted to a workflow step.)

Reply:
Oh, that's right; that's annoying. At some point I'd like to be able to pass in a logger function to the classifier so it can be handled by the consuming app.

Basically, I'd want sentry to log an error if we have a project running on the classifier that it can't support, but that's outside this scope.

Reply:
That's come up with feedback too, where we'd like to report if there's a mistake in the subject metadata that breaks feedback, without completely crashing the classifier.